src/lua/zencode.lua

```diff
@@ -636,7 +637,9 @@ function zencode:run()
 	end
 
 	-- EXEC zencode
-	for _, x in pairs(self.AST) do
+	while ZEN.next_instruction <= #self.AST do
+		x = self.AST[ZEN.next_instruction]
```

Comment:
This is OK. Just a note, perhaps TODO: for optimization, to develop a lua iterator, which would save lookup time
https://www.lua.org/pil/7.1.html